AddOns/Elster/app/src/Pages/PAG11016.SalesVATAdvNotifCard.al

```diff
@@ -116,11 +116,6 @@ page 11016 "Sales VAT Adv. Notif. Card"
                     ApplicationArea = Basic, Suite;
                     ToolTip = 'Specifies the VAT statement template name used to calculate the amounts and assign them to the key figures required by the tax authorities.';
                 }
-                field("Use Authentication"; "Use Authentication")
```

Comment:
Breaking Change. Field should get deprecated.

Reply:
Sorry for delay. 
Do you mean that I should not have removed the "Use Authentication" field from the page?
Well I did this because of this compiler error message:
> Field 'Use Authentication' is removed. Reason: Authentication is required for electronic transmission.AL(AL0433)

I initially tried to keep it at the page and simply add an ObsoleteReason.
However, I changed the ObsoleteState to Pending and modifed the ObsoleteReason. Further I removed the Use Authentication field from Tests. I think more changes are not necessary referring to my changes.